lms/templates/registration/password_reset_confirm.html

```diff
@@ -3,148 +3,72 @@
-            {% if err_msg %}
-              <li>{{err_msg}}</li>
+<div id="password-reset-confirm-container" class="login-register">
+    <section id="password-reset-confirm-anchor" class="form-type">
+        <div id="password-reset-confirm-form" class="form-wrapper" aria-hidden="true">
+            <div class="status submission-error {% if not err_msg %} hidden {% endif %}" aria-live="polite">
+                <h4 class="message-title">{% trans "The following errors occurred while processing your registration: " %}</h4>
+                <ul class="message-copy">
+                {% if err_msg %}
+                    <li>{{err_msg}}</li>
+                {% else %}
+                    <li>{% trans "You must complete all fields." %}</li>
+                    <li>{% trans "The two password fields didn't match." %}</li>
+                {% endif %}
+                </ul>
+            </div>
+            {% if validlink %}
+
+            <form role="form" id="passwordreset-form" method="post" action="">{% csrf_token %}
+                <div class="section-title lines">
+                    <h2>
+                        <span class="text">
+                        {% blocktrans with platform_name=platform_name %}
+                            Reset your {{ platform_name }} password
+                        {% endblocktrans %}
+                        </span>
+                    </h2>
+                </div>
+
+                <p class="action-label">
+                    {% trans "Please enter your new password twice so we can verify you typed it in correctly." %}
+                </p>
+
+                <div class="form-field new_password1-new_password1">
+                    <label for="new_password1">{% trans "Your new password" %}</label>
+                    <input id="new_password1" type="password" name="new_password1" placeholder="*****" />
+                </div>
+                <div class="form-field new_password2-new_password2">
+                    <label for="new_password2">{% trans "Your new password again" %}</label>
+                    <input id="new_password2" type="password" name="new_password2" placeholder="*****" />
+                </div>
+
+                <button type="submit" class="action action-primary action-update js-reset">{% trans "Change my password" %}</button>
+            </form>
             {% else %}
-              <li>{% trans "You must complete all fields." %}</li>
-              <li>{% trans "The two password fields didn't match." %}</li>
+            <div class="status submission-error" aria-live="polite">
+                <h4 class="message-title">{% trans "The password reset link was invalid" %}</h4>
+                <ul class="message-copy">
+                {% blocktrans with start_link='<a href="/login">' end_link='</a>' %}
+                    Possibly because the link has already been used. Please return to the {{ start_link }}login page{{ end_link }} and start the password reset process again.
```

Comment:
This password reset link is invalid. It may have been used already. To reset your password, go to the {{ start_link }}sign-in{{ end_link }} page and select **Reset Password**.

(bold "Reset Password")